src/lib/containers/entity_finder/Breadcrumbs.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+
+export type BreadcrumbItem = {
+  name: string
+  isCurrent: boolean
+  action?: () => void
+}
+
+export type BreadcrumbsProps = {
+  items: BreadcrumbItem[]
+  maxItemCount?: number
+}
+
+export const Breadcrumbs: React.FunctionComponent<BreadcrumbsProps> = ({
```

Comment:
Might take some refactoring (esp. styling) but this should be reusable for any type of breadcrumb